proto/netbookai/spawner/spawner.proto

```diff
@@ -84,6 +84,15 @@ service SpawnerService {
 
   rpc RegisterClusterOIDC(RegisterClusterOIDCRequest)
       returns (RegisterClusterOIDCResponse) {}
+
+  rpc AppendRoute53Records(AppendRoute53RecordsRequest)
```

Comment:
Let's call this `AddRoute53Records` ? And similar changes to request and response objects?

Reply:
I used the same names as were expected by that DNS interface, we can do it Add*, Append* is not bad

Reply:
Append gives the sense that we are adding value to an existing record.